package.json

```
@@ -33,6 +34,7 @@
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.1",
+    "@adobe/helix-fetch": "^1.6.1",
```

```suggestion
    "@adobe/helix-fetch": "1.6.1",
```
Use pinned dependencies